### PR TITLE
OPSEXP-1353 Removed Alfresco Sync service HPA chart

### PR DESCRIPTION
**Pull request by upandey123**
Sync service is the only Chart that has some kind of autoscaling.

It is untested and leverage a now depreceated approach:

 

autoscaling/v2beta1 HorizontalPodAutoscaler is deprecated in v1.22+, unavailable in v1.25+; use autoscaling/v2beta2 HorizontalPodAutoscaler
Remove it from the subchart

### CODE REVIEW

File: helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml [hunk 1]
```diff
@@ -1,33 +0,0 @@
-{{- if .Values.syncservice.enabled }}
-{{- if .Values.syncservice.horizontalPodAutoscaling.enabled }}
```
Comment (alxgomz) [inline helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml:2]: These values need to be removed from the `values.yaml` file

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml (hunk 1 above)
Comment (upandey123) [reply helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml:2]: Removed {{- if .Values.syncservice.enabled }} but syncservice.horizontalPodAutoscaling.enabled is not inside values file.

**Review by upandey123 — COMMENTED** [review]
Removed {{- if .Values.syncservice.enabled }} but syncservice.horizontalPodAutoscaling.enabled is not inside values file.

**Comment by alxgomz** [comment]
> Removed {{- if .Values.syncservice.enabled }} but syncservice.horizontalPodAutoscaling.enabled is not inside values file.

No, you're looking at the wrong file. do not edit the `helm/alfresco-content-services/values.yaml` instead you should edit `helm/alfresco-content-services/charts/alfresco-sync-service/templates/hpa-syncservice.yaml`

**Comment by upandey123** [comment]
As checked from my end all the HPA resources have been removed from the sync service. Should be good!

File: helm/alfresco-content-services/charts/alfresco-sync-service/values.yaml [hunk 2]
```diff
@@ -15,7 +15,6 @@ podSecurityContext:
   runAsNonRoot: true
   fsGroup: 1000
 syncservice:
-  enabled: true
```
Comment (gionn) [inline helm/alfresco-content-services/charts/alfresco-sync-service/values.yaml:18]: it's still used in the [deployment](https://github.com/Alfresco/acs-deployment/blob/master/helm/alfresco-content-services/charts/alfresco-sync-service/templates/deployment-syncservice.yaml#L1)

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]
LGTM

I will merge it since codacy is broken again.

**Pull request merged**